packages/astro/components/Code.astro

```diff
@@ -89,7 +89,7 @@ const highlighter = await getCachedHighlighter({
 
 const html = highlighter.highlight(code, typeof lang === 'string' ? lang : lang.name, {
 	inline,
```

Comment:
For reviewers, those cast look suspicious, but there's actually already a bunch of said casts in the file. I don't really know why, perhaps @bluwy has more context having worked on Shiki recently?

Reply:
Okay, not sure what happened, GitHub moved my comment.. but I'm talking about the `as any` casts on `lang`.